### PR TITLE
Devnet RESET D1.5.13.1

### PR DESCRIPTION
**Pull request by iulianpascalau**
- devnet reset configs D1.5.13.1
Genesis is set to timestamp 1694000000 which will happen on September 6th, 2023, 11:33:20 UTC

### CODE REVIEW

**Review by valentin-lup — DISMISSED** [review]

File: seednodes/p2p.toml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    #InitialPeerList represents the list of strings of some known nodes that will bootstrap this node
+    #The address will be in a self-describing addressing format.
+    #More can be found here: https://github.com/libp2p/specs/blob/master/3-requirements.md#34-transport-agnostic
+    #Example:
+    #   /ip6/fe80::8823:6dff:fee7:f172/tcp/4001/p2p/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #   /ip4/162.246.145.218/udp/4001/utp/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #
+    #If the initial peers list is left empty, the node will not try to connect to other peers during initial bootstrap
+    #phase but will accept connections and will do the network discovery if another peer connects to it
+    InitialPeerList = []
```
Comment (AdoAdoAdo) [inline seednodes/p2p.toml:59]: add the same seeders as in node p2p toml?

File: seednodes/p2p.toml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    #kademlia's routing table bucket size
+    BucketSize = 100
+
+    #RoutingTableRefreshIntervalInSec defines how many seconds should pass between 2 kad routing table auto refresh calls
+    RoutingTableRefreshIntervalInSec = 300
+
+[Sharding]
```
Comment (AdoAdoAdo) [inline seednodes/p2p.toml:67]: this could be removed, the seed node does not need to care about sharding

File: systemSmartContractsConfig.toml [hunk 3]
```diff
@@ -13,11 +13,11 @@
     ActivateBLSPubKeyMessageVerification = true
 
 [ESDTSystemSCConfig]
-    BaseIssuingCost = "50000000000000000" #0.05 eGLD
-    OwnerAddress = "erd1uq2tmp2aeev3wdl2znphchenztngjpc030ze46c38wqru4vf5gss4r6kuk"
+    BaseIssuingCost = "5000000000000000000" #5 eGLD
```
Comment (AdoAdoAdo) [inline systemSmartContractsConfig.toml:16]: 0.05 egld?

File: nodesSetup.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "startTime": 1648551600,
+  "startTime": 1693317600,
```
Comment (AdoAdoAdo) [inline nodesSetup.json:2]: if we don't release it in time this needs to be changed.

File: seednodes/p2p.toml (hunk 1 above)
Comment (iulianpascalau) [reply seednodes/p2p.toml:59]: done

File: seednodes/p2p.toml (hunk 2 above)
Comment (iulianpascalau) [reply seednodes/p2p.toml:67]: added task MX-14498

File: systemSmartContractsConfig.toml (hunk 3 above)
Comment (iulianpascalau) [reply systemSmartContractsConfig.toml:16]: right, reverted

**Review by bogdan-rosianu — APPROVED** [review]

File: nodesSetup.json (hunk 4 above)
Comment (iulianpascalau) [reply nodesSetup.json:2]: right, this is set for the dry-run

**Review by bogdan-rosianu — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by valentin-lup — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**